src/tests/ftest/util/dmg_utils.py

```diff
@@ -321,6 +321,45 @@ def storage_query_list_pools(self, uuid=None, rank=None, verbose=False):
             ("storage", "query", "list-pools"), uuid=uuid, rank=rank,
             verbose=verbose)
 
+    def storage_identify_device(self, uuid=None,  verbose=False):
+        """Get the result of the 'dmg storage query list-pools' command.
+
+        Args:
+            uuid (str): Device UUID to query. Defaults to None.
+            verbose (bool, optional): create verbose output. Defaults to False.
+
+        Returns:
+            CmdResult: an avocado CmdResult object containing the dmg command
+                information, e.g. exit status, stdout, stderr, etc.
+
+        Raises:
+            CommandFailure: if the dmg storage query command fails.
+
+        """
+        return self._get_result(
+            ("storage", "identify", "vmd" ), uuid=uuid,
+            verbose=verbose)
+
+    def storage_replace_device(self, old_uuid=None,  new_uuid=None, no_reint=False):
```

Comment:
These methods should match the command.  Shouldn't this be `storage_replace_nvme`?

Reply:
Done.